src/utils.js

```diff
@@ -104,7 +104,7 @@ function processEnv(config) {
   }
 
   config.computed.ci.isPr = config.computed.ci.prNumber !== 'false'
-  config.computed.ci.branch = getEnv(config.ci.env.branch, 'master')
+  config.computed.ci.branch = getEnv(config.ci.env.branch, config.vcs.repository.mainBranch)
```

Comment:
Might want to mention this in the changelog

Reply:
It's actually not used unless there's no branch env variable in `ci`, but still worth mentioning. Also, I think. Messed up, I was gonna call it `defaultBranch` instead of `mainBranch`. I'll fix that.